src/TreeView/useRovingTabIndex.ts

```diff
@@ -30,20 +30,26 @@ export function getNextFocusableElement(activeElement: HTMLElement, event: Keybo
       // Close node; don't change focus
       return
 
+    // Focus parent element
+    case 'open Backspace':
+      return getParentElement(activeElement)
+
     case 'closed ArrowRight':
       // Open node; don't change focus
       return
 
+    // Focus parent element
     case 'closed ArrowLeft':
-      // Focus parent element
+    case 'closed Backspace':
       return getParentElement(activeElement)
 
     case 'end ArrowRight':
       // Do nothing
       return
 
+    // Focus parent element
     case 'end ArrowLeft':
-      // Focus parent element
+    case 'end Backspace':
       return getParentElement(activeElement)
```

Comment:
You could also add `Backspace` to the switch statement below this on line 50 since the behavior of `Backspace` doesn't depend on the state of the node